static/app/components/replays/replayContext.tsx

```diff
@@ -11,6 +12,14 @@ import useRAF from 'sentry/utils/replays/hooks/useRAF';
 import type ReplayReader from 'sentry/utils/replays/replayReader';
 import usePrevious from 'sentry/utils/usePrevious';
 
+enum ReplayLocalstorageKeys {
+  ReplayConfig = 'replay-config',
+}
```

Comment:
What is the benefit of making this an enum? Could it be a const instead?

Reply:
I don't think there's a direct benefit of it I just tend to use enums for type safety when it comes to these cases as I found easier to mantain/debug. But it's a personal preference, I could switch it to a const tbh, which I think will be fine as well!